test/plugin/base_test.rb

```diff
@@ -1762,9 +1834,8 @@ def setup_logging_stubs
     _undefined
   end
 
-  # Verify the number and the content of the log entries match the expectation.
-  # The caller can optionally provide a block which is called for each entry.
-  def verify_log_entries(_n, _params, _payload_type = 'textPayload', &_block)
+  # Jsonify the log entries response if it is not in JSON format.
```

Comment:
Careful -- this is not about the response, this is about the ingested entries themselves.